### PR TITLE
AttackSurfaceReduction

### PR DESCRIPTION
**Pull request by DanielChronlund**
Change(s):
   - Added a new workbook called 'AttackSurfaceReduction'.

   Reason for Change(s):
   - This workbook helps you implement the ASR rules of Windows/Defender, and to monitor them over time. The workbook can filter on rules in Audit mode and Block mode. Please see details on my blog: https://danielchronlund.com/2022/06/15/attack-surface-reduction-dashboard-for-microsoft-sentinel/

   Version Updated:
   - No

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

File: Workbooks/WorkbooksMetadata.json [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+    "logoFileName": "M365securityposturelogo.svg",
+    "description": "This workbook helps you implement the ASR rules of Windows/Defender, and to monitor them over time. The workbook can filter on ASR rules in Audit mode and Block mode.",
+    "dataTypesDependencies": [ "DeviceEvents"],
+    "dataConnectorsDependencies": [ "MicrosoftThreatProtection" ],
+    "previewImagesFileNames": [ "AttackSurfaceReductionWhite.png", "AttackSurfaceReductionBlack.png" ],
+    "version": "1.0.0",
+    "title": "Attack Surface Reduction Dashboard",
+    "templateRelativePath": "AttackSurfaceReduction.json",
+    "subtitle": "",
+    "provider": "Daniel Chronlund"
```
Comment (v-sabiraj) [inline Workbooks/WorkbooksMetadata.json:13]: Hey @DanielChronlund, provider should be name of organization, can you please check and update, thanks.

Comment (DanielChronlund) [reply Workbooks/WorkbooksMetadata.json:13]: @v-sabiraj, this is a private MVP contribution. Is a company name required as the provider?

Comment (v-sabiraj) [reply Workbooks/WorkbooksMetadata.json:13]: @DanielChronlund, you can add as Microsoft or Microsoft Sentinel community

Comment (v-sabiraj) [reply Workbooks/WorkbooksMetadata.json:13]: > @DanielChronlund, you can add as Microsoft or Microsoft Sentinel community

**Comment by v-sabiraj** [comment]
Hey @DanielChronlund, can you please quickly change this, so we can proceed to merge, thanks.

**Comment by DanielChronlund** [comment]
Yes, I'm currently travelling but I will change this first thing tomorrow
Friday.

Den tors 8 sep. 2022 08:18v-sabiraj ***@***.***> skrev:

**Comment by NikTripathi** [comment]
Thanks @DanielChronlund Please let us know once done. Thanks

**Review by v-sabiraj — APPROVED** [review]

**Review by NikTripathi — APPROVED** [review]

**Pull request merged**